hw/application_fpga/core/tk1/toolruns/Makefile

```diff
@@ -18,7 +18,7 @@ CC = iverilog
 CC_FLAGS = -Wall
 
 LINT = verilator
```

Comment:
Should `udi_rom.v` be linted or simlutated as well? It does not seem to be included now.

Reply:
Conversation with Joachim, this is done separately.